src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -19,6 +19,7 @@
  * @see tribe_get_event() For the format of the event object and its properties.
  *
  */
+use Tribe__Date_Utils as Date;
```

Comment:
For consistency with what we do in other code fragments, and to avoid confusion, change the alias to `Dates` (plural).